specifyweb/specify/urls.py

```diff
@@ -32,7 +32,7 @@
         url(r'^(?P<id>\d+)/desynonymize/$', tree_views.desynonymize),
         url(r'^(?P<parentid>\d+)/predict_fullname/$', tree_views.predict_fullname),
         url(r'^(?P<treedef>\d+)/(?P<parentid>\w+)/stats/$', tree_views.tree_stats),
-        url(r'^(?P<treedef>\d+)/(?P<parentid>\w+)/(?P<sortfield>\w+)/$', tree_views.tree_view),
+        url(r'^(?P<treedef>\d+)/(?P<parentid>\w+)/(?P<sortfield>\w+)/(?P<includeAuthor>\w+)$', tree_views.tree_view),
```

Comment:
Could you also define OpenAPI schema for this endpoint?
Others too if you have time

Reply:
Fixed in https://github.com/specify/specify7/commit/b4a084f0cfdb2c133782f709cb8be62e5cef98a0